packages/palette/src/elements/Input/Input.tsx

```diff
@@ -44,30 +47,24 @@ export const Input: React.ForwardRefExoticComponent<
   ) => {
     const [boxProps, inputProps] = splitBoxProps(rest)
 
+    const inputName = inputProps.name || "palette-input"
+
     return (
       <Box width="100%" className={className} {...boxProps}>
-        {(title || description) && (
-          <>
-            {title && (
-              <Text variant="xs">
-                {title}
-                {required && (
-                  <Box as="span" color="brand">
-                    *
-                  </Box>
-                )}
-              </Text>
-            )}
-
-            {description && (
-              <Text variant="xs" color="black60">
-                {description}
-              </Text>
-            )}
-          </>
+        {description && (
+          <Tooltip
+            pointer
+            content={description}
+            variant={descriptionVariant}
+            placement={"top-end"}
+          >
+            <Text variant="xs" color="black60" textAlign="right">
+              <u>What is this?</u>
+            </Text>
```

Comment:
👍 The `Tooltip` component clones this child and adds `tabIndex={0}` to ensure this is keyboard focusable.